chapters/02-estrutura-do-programa.md

```diff
@@ -345,9 +345,11 @@ Usar o operador resto (`%`) é uma maneira fácil de testar se um número é div
 
 A construção do `for` nesse exemplo não contém a parte que checa pelo fim do loop. Isso significa que o loop não vai parar de executar até que a declaração `break` contida nele seja executada.
 
-Se a declaração `break` faltar, ou acidentalmente tivermos uma condição que sempre produza `true`, você terá o chamado *loop infinito*. Um programa rodando um loop infinito nunca vai parar de rodar, que é normalmente uma coisa ruim.
+Se você não incluir a declaração `break` ou acidentalmente escrever uma condição que sempre produza um resultado `true`, seu programa ficará preso em um _loop infinito_. Um programa preso em um loop infinito nunca vai terminar sua execução, o que normalmente é uma coisa ruim.
```

Comment:
eu reparei que o autor introduz o itálico para destacar conceitos novos e importantes toda vez que eles são citados pela primeira vez, por isso mantive o itálico igual ao original. eu tenho a impressão de que isso pode ser benéfico ao leitor, justamente por destacar esses conceitos importantes, o que acha? você tem alguma outra sugestão de destaque ou simplesmente sugere que removemos ele? //cc @ericdouglas